src/ui/preferences/GeneralSettings.qml

```diff
@@ -498,6 +498,7 @@ QGCView {
 
                                     model:  ListModel {
                                         ListElement { text: "disabled" }
+                                        ListElement { text: "UDP Port"}
```

Comment:
There is no option for serial here... but also before the patch?

Reply:
Of course it lists the serial devices connected (if any).

Reply:
Yeah, it is populated dinamically

Reply:
done!